lib/stack/prisma-lambda-cdk-stack.ts

```diff
@@ -25,7 +25,7 @@ export class PrismaLambdaCdkStack extends cdk.Stack {
         // You should create a database user with minimal privileges for your application.
         // Also refer to: https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/UsingWithRDS.MasterAccounts.html
         username: database.secret.secretValueFromJson("username").toString(),
-        password: database.secret.secretValueFromJson("password").toString(),
+        password: encodeURIComponent(database.secret.secretValueFromJson("password").toString()),
```

Comment:
Ideally, I think this would be encoded only when the URL is constructed (i.e. when building `DATABASE_URL` in https://github.com/aws-samples/prisma-lambda-cdk/blob/main/backend/prisma/.env) -- but safely encoding URL components in bash is beyond my talents